app/v1/connectors/BaseDesConnector.scala

```diff
@@ -28,12 +28,16 @@ trait BaseDesConnector {
   val http: HttpClient
   val appConfig: AppConfig
 
-  val logger = Logger(this.getClass)
+  val logger: Logger = Logger(this.getClass)
 
   private[connectors] def desHeaderCarrier(implicit hc: HeaderCarrier): HeaderCarrier =
     hc.copy(authorization = Some(Authorization(s"Bearer ${appConfig.desToken}")))
       .withExtraHeaders("Environment" -> appConfig.desEnv)
 
+  private[connectors] def desPostHeaderCarrier(implicit hc: HeaderCarrier): HeaderCarrier =
+    hc.copy(authorization = Some(Authorization(s"Bearer ${appConfig.desToken}")))
+      .withExtraHeaders("Environment" -> appConfig.desEnv, "Content-Type" -> "application/json", "OriginatorID" -> "SCAN")
```

Comment:
the Content-Type header should already be included with the user request so doesn't need to be added here as well. Also, we are missing correlationId here, as per https://github.com/hmrc/vat-api/blob/58f215c7452848d8be93f4b2361c747c7e707652/app/v1/connectors/BaseDesConnector.scala#L42